dev-docs/modules/userId.md

```diff
@@ -69,6 +69,54 @@ of sub-objects. The table below has the options that are common across ID system
 
 ## User ID Sub-Modules
 
+### AdmixerID
+
+Admixer ID, provided by [Admixer](https://admixer.com/), is a universal ID solution that doesn’t rely on 3rd party cookies and gives publishers access to a wide range of Admixer’s demand sources, amplifying audience segmentation, targeting and measurement.
+This sub-module enables the Admixer ID to be available in the bid request.
+
+The Admixer privacy policy is at https://admixer.com/privacy/ 
+
+Add Admixer ID module to your Prebid.js package with:
+
+{: .alert.alert-info :}
+gulp build --modules=admixerIdSystem
+
+#### AdmixerID Configuration
+
+{: .table .table-bordered .table-striped }
+| Param under userSync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| name | Required | String | `"admixerId"` | `"admixerId"` |
+| params | Required | Object | Details for admixer initialization. | |
+| params.pid | Required | String | id provided by admixer | "458frgde-djd7-3ert-gyhu-12fghy76dnmko" |
+| params.e | Optional | String | The hashed email address of a user. We can accept the hashes, which use the following hashing algorithms: md5, sha1, sha256. | "3d400b57e069c993babea0bd9efa79e5dc698e16c042686569faae20391fd7ea" |
+| params.p | Optional | String | The hashed phone number of a user. We can accept the hashes, which use the following hashing algorithms: md5, sha1, sha256. | "05de6c07eb3ea4bce45adca4e0182e771d80fbb99e12401416ca84ddf94c3eb9" |
+
+#### AdmixerID Examples
+
+1) Individual params may be set for the Admixer ID Submodule.
+
+{% highlight javascript %}
+   pbjs.setConfig({
+       userSync: {
+           userIds: [{
+               name: "admixerId",
+               storage: {
+                   name: "__adm__admixer",
+                   type: "cookie",
+                   expires: 30
+               },
+               params: {
+                   pid: "4D393FAC-B6BB-4E19-8396-0A4813607316", // example id
+                   e: "3d400b57e069c993babea0bd9efa79e5dc698e16c042686569faae20391fd7ea", // example hashed email (sha256)
```

Comment:
Is the Publisher responsible for resolving and hashing the users email address? Same question for the line below.

Reply:
yes, is that a problem ?

Reply:
not per se, but this functionality already exists in the PubProvided module https://docs.prebid.org/dev-docs/modules/userId.html#pubprovided-id. That module takes as input any publisher defined function as long as it generates an eids style object.  

Your module requires a pub function to grab and hash an email or phone #. Instead of passing that value to the admixer module, why not have that function pass an eids style object to the pubprovided module?

Reply:
i'd prefer leave it the way as is.  i think that setup through submodule params is more clearly for publisher.

Reply:
Ok that is fine with me.  Please clearly describe what you are asking publishers to do in your sub adapter description. There should be explicit mention of hashed email and phone # in the description

Reply:
we described it in config section.

| params.e | Optional | String | The hashed email address of a user. We can accept the hashes, which use the following hashing algorithms: md5, sha1, sha256. | "3d400b57e069c993babea0bd9efa79e5dc698e16c042686569faae20391fd7ea" |

| params.p | Optional | String | The hashed phone number of a user. We can accept the hashes, which use the following hashing algorithms: md5, sha1, sha256. | "05de6c07eb3ea4bce45adca4e0182e771d80fbb99e12401416ca84ddf94c3eb9" |

it's optional param, this is not enough ?

Reply:
Correct, we'd like to see something a little more descriptive. Something along the lines of 
```
Admixer ID, provided by [Admixer](https://admixer.com/), is a universal ID solution that doesn’t rely on 3rd party cookies.   Our sub adapter takes deterministic signals like email and phone as input and returns an anonymous id that unlocks access to a wide range of Admixer’s demand sources, amplifying audience segmentation, targeting and measurement.
```
Or something similar, but you need to clearly state in the description that your sub adapter takes PII as input.  That is not something other adapters do.

Reply:
updated